backend/src/core/constants.ts

```diff
@@ -17,6 +17,7 @@ export enum MessageStatus {
   Sending = 'SENDING',
   Error = 'ERROR',
   Success = 'SUCCESS',
+  Read = 'READ',
```

Comment:
Was there a need to manually add `READ` as a possible value in `enum_email_messages_status`? If so, would be good to add a database migration to do this.

Reply:
I added it here as the `email_messages` table `status` field references `MessageStatus` for the allowed enum values. Is that what you mean?
https://github.com/opengovsg/postmangovsg/blob/54d46e6e0b7f80b34a5cd5fcacdec0e9c554c876/backend/src/email/models/email-message.ts#L43-L47

Reply:
Since we don't use `sequelize.sync` anymore, I'm not sure if this will automatically update the Postgresql enum named `enum_email_messages_status`. We might have to manually modify the enum by running:

```sql
ALTER TYPE enum_email_messages_status ADD VALUE 'READ';
```

This should be added as a sequelize database migration under `backend/src/database`.